site/RoomDraw/src/app/authentication/authentication.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthenticationService {
+  isLoggedIn = false;
+
+  constructor() {}
+
+  login() {
+    // Could API calls go here?
```

Comment:
Yes they can go there.  Can you add those before I merge them?  There's info on how they work on teams.